setup.py

```diff
@@ -39,5 +39,5 @@
           packages=find_packages(),
           package_data={'dirty_cat': ['VERSION.txt', 'data/midwest_survey/*.csv']},
           install_requires=['scikit-learn>=0.20', 'numpy', 'scipy', 'requests',
-                            'joblib'],
+                            'joblib', 'fasttext>=0.9.2'],
```

Comment:
Can we make it so that fasttext is an optional dependency: it is loaded only inside the methods of the class that need it.

Reply:
I can move `import fasttext` inside the methods that use it. Regarding `setup.py`, should I declare optional dependencies (see below) or just remove fasttext from `install_requires` ?
```
extras_require={
              "fasttext": ["fasttext>=0.9.2"]  
            }
```

If we declare optional dependencies with `extras_require`, then should we add `python-Levensthein` too ? It is presented as an optional dependency in `README.rst` but isn't mentioned in `setup.py`.